### PR TITLE
[Wicked] Add Bridge interfaces from Wicked XML files

### PR DESCRIPTION
**Pull request by jlausuch**
- Related ticket: https://progress.opensuse.org/issues/36178
- Verification run: http://fromm.arch.suse.de/tests/973#step/advanced_sut/747

### CODE REVIEW

**Comment by jlausuch** [comment]
@asmorodskyi @mtomaschewski @rtorrero
With this 5 more commits, we can conclude the test cases we wanted to show for the fist demo of the advanced suite.

**Comment by rtorrero** [comment]
Thanks @jlausuch! Nice to see all the progress. One thing I noticed: commit 07b7ddccde2d2fe4cfbd6f513d80fb0fbcdf95b5 mentions ifcfg files but it seems to me that it is referencing XML files instead. Was that a mistake or am I missing something?

**Comment by jlausuch** [comment]
@rtorrero That's strange, when you click on it, the title is "[Wicked] Add SIT interface from Wicked XML files"
 https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5440

That might be cause I changed the tillte directly in github .... I don't think it's a big deal

File: tests/wicked/advanced_sut.pm [hunk 1]
```diff
@@ -12,11 +12,11 @@
...
-# Test 4 : Create a SIT interface from Wicked XML files [NOT IMPLEMENTED]
+# Test 4 : Create a SIT interface from Wicked XML files
 # Test 5 : Create a IPIP  interface from legacy ifcfg files
-# Test 6 : Create a IPIP interface from Wicked XML files [NOT IMPLEMENTED]
+# Test 6 : Create a IPIP interface from Wicked XML files
 # Test 7 : Create a tun interface from legacy ifcfg files
-# Test 8 : Create a tun interface from Wicked XML files [NOT IMPLEMENTED]
+# Test 8 : Create a tun interface from Wicked XML files
 # Test 9 : Create a tap interface from legacy ifcfg files
 # Test 10: Create a tap interface from Wicked XML files [NOT IMPLEMENTED]
```
Comment (asmorodskyi) [inline tests/wicked/advanced_sut.pm]: this one also implemented

**Pull request merged**